packages/react/src/drafts/IssueLabel/getColorFromHex.ts

```diff
@@ -0,0 +1,138 @@
+import {getContrast} from 'color2k'
+import {Hsluv} from 'hsluv'
+
+/**
+ * transforms a hex color provided by the user into a color object with background and text colors
+ * @param colorHex — the hex color provided by the user
+ * @param colorScheme — the color scheme the user has currently set
+ * @param isSelected
+ * @param bgColor — the background color from the selected theme, needed to calc the contrast for the colors
+ * @returns
+ */
+export const getColorsFromHex = (
+  colorHex: string,
+  colorScheme = 'light',
```

Comment:
Is there a union type for this anywhere? Would be great to use that (or construct our own) if possible

Reply:
I don't think so, at least I did not find any in the ThemeProvider